### ISSUE TITLE
[ADE 508 Audit]: Office of Section 508 Compliance: My VA (44409): 508 Audit Report Findings Support

### ISSUE THREAD

**Issue by lakper** [department-of-veterans-affairs/va.gov-team#115221]
> [!NOTE]
> **QUICK SUMMARY:** _Parent ticket created to track Section 508 audit findings for the MHV Home Page as part of Top Ten INTERnet Websites - June 2025 VASI._

## 508 Audit Office Scoring

**Overall Score:** `34`
- Critical: `4`
- High: `1`
- Medium: `0`
- Low: `1`
- Total: `0`

### About the 508 Office's Scoring

**A product with no defects has a score of 100**

- Each open 'Critical' defect lowers the score by 15.
- Each open 'High' defect lowers the score by 5.
- Each open 'Medium' defect lowers the score by 2.5.
- Each open 'Low' defect lowers the score by 1.
- Accessibility defects have no effect on the score.

### List of Defects:
The following examples are not and should not be interpreted as an exhaustive or complete list of all defects found in the software product. Examples were chosen to ease reproduction of the accessibility defect and to illustrate where multiple defects produce additional barriers to accessibility in combination. Stakeholders of the product should conduct internal testing to identify any additional instances of defects identified in this report. However, because these examples list the end-user experience of accessibility, defects identified may result from or be explained in terms of multiple criteria.
 
### Defect 32 | Critical | Created:
_WCAG 1.3.1 Information and Relationships | Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text._

My VA | Veterans Affairs screen (Here’s how you know)
On the My VA | Veterans Affairs screen, when Here’s how you know link is activated, JAWS does not announce “The .gov means it’s official” or “The site is secure,” text as a heading. This does not allow the structure conveyed by headings to be understood as conveyed through presentation. This text is deemed to be headings based on the following criteria: font weight, text size and semantic relation to other content. (New: 06/24/2025)

### Defect 33 | Critical | Created:
_WCAG 1.3.1 Information and Relationships | Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text._

My VA | Veterans Affairs screen, footer region
The list structure in the footer of the screen cannot be understood by assistive technology. When navigating with arrow keys “list of 5 items” is read for the content following Get answers. However, when navigating to list items headings are included in that list while links such as the phone number is not a listed item. This does not clearly convey the structure of information presented. Please see attachment. (New: 06/24/2025)

### 3. For TTY, Call 711 then 988 is not read entirely by screen reader
**Defect 34 | Critical | Created:**
_WCAG 1.3.1 Information and Relationships | Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text._

Talk to the Veterans Crisis Line Now Modal
When focus lands on the ‘For TTY, call 711 then 988’ link. What is read by screen reader is not the full text that is in the modal. All that is announced by screen reader is: “711 link, to activate press enter.” The extension number/important info is not read out to the user. (New: 06/24/2025)

### Defect 35 | Low | Created:
_WCAG 2.4.2 Page Titled | Web pages have titles that describe topic or purpose._

My VA | Veterans Affairs screen (Talk to the Veterans Crisis Line now)
The title of the modal dialog does not describe the topic of modal. For example, this happens when user opens Talk to the Veterans Crisis Line now modal dialog. Jaws announces title as “My VA | Veterans Affairs,” this does not describe the topic or purpose of the dialog modal. Additionally, when user has been idle for a time a modal pop-up alerting user their session will end in a couple of seconds. The title of the modal is also the same as the main page. Please see attachment. (New: 06/24/2025)

### Defect 36 | High | Created:
_WCAG 2.4.3 Focus Order | If a Web page can be navigated sequentially and the navigation sequences affect meaning or operation, focusable components receive focus in an order that preserves meaning and operability._

Throughout the product. Example on Feedback Survey Modal dialog (Feedback)
Throughout the product, focus does not follow a logical order. When keyboard user navigates the Feedback Survey Modal dialog using tab key, the Close button at the bottom of the modal dialog receives focus before the Submit button that is structurally placed before it. These buttons in the content do not receive focus in an order that preserves meaning. Please see attachment. (New: 06/24/2025)
Insufficient focus indicator

### 6. Defect 37 | Critical | Created:
_WCAG 2.4.7 Focus Visible | Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible._
My VA | Veterans Affairs screen
Throughout the product, elements do not have sufficiently visible indication of focus. When keyboard user navigates around My VA | Veterans Affairs screen, the color contrast between the background and focus indicator is 1.6:1 which is not sufficient contrast to indicate focus. Please see attachment. (New: 06/24/2025)

### Project Background

- Project Name: Top Ten INTERnet Websites - June 2025 VASI #: None  
- Audit Number: 44409 Audit Date: 6/24/2025
- Audit Scope: My VA
- Product Type: Websites
- Project Status: Open
- Project Site: https://www.va.gov/my-va
- Project Version: 06/24/2025
- Release Recommendation: No
- Release Recommendation Comments: Product is non-conformant
- Link to 508 Compliance Application: [508 Compliance Application](https://apps.gov.powerapps.us/play/e/default-e95f1b23-abaf-45ee-821d-b7ab251ab3bf/a/5f7da63e-384d-4ce4-8cc7-f3be6c60c363?tenantId=e95f1b23-abaf-45ee-821d-b7ab251ab3bf&sourcetime=2023-08-14%2017%3A50%3A55Z&RecordID=2989)

### Ticket Updates
- [ ] _Connect to an `Epic/Parent` or `Intake` (what body of work is this a part of?) If this is the `Epic/Parent`, connect related tickets as sub-issues._
- [ ] _If this is the `Epic/Parent` ticket, add label for `ADE 508 RECHECK`_
- [ ] _If this ticket is a `Child` ticket aka `sub-issue` please add the appropriate labels for `ADE 508 FLAG CRITICAL`;`ADE 508 FLAG HIGH`; `ADE 508 FLAG MEDIUM` or `ADE 508 FLAG LOW`_
- [ ] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [ ] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [ ] _Select a `Priority Level`_
- [ ] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
- [ ] _Make sure the `ADE 508 RECHECK` label is on this ticket upon closing and that a follow-up date has been established and calendared._

**Comment by artsymartha68** [comment]
Another update on issue 5 per #58574 : fixed now!

> I’ve reverted the buttons to Medallia’s default order so the visual order now matches the keyboard/tab order.
> 
> This change aligns the visual button order with the keyboard/tab order, meeting WCAG 2.4.3 Focus Order. It ensures that users navigating with a keyboard or assistive technology encounter the buttons in a logical and predictable sequence.
